apps/src/code-studio/components/progress/StageProgress.jsx

```diff
@@ -11,14 +11,17 @@ import { levelType } from '@cdo/apps/templates/progress/progressTypes';
 const styles = {
   headerContainer: {
     // With our new bubble we don't want any padding above/below
-    paddingTop: 0,
-    paddingBottom: 0,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
```

Comment:
I think it doesnt matter in this case because we never have more space than content